package.json

```diff
@@ -43,7 +44,7 @@
     "base-x": "^3.0.5",
     "basic-auth": "^2.0.1",
     "body-parser": "^1.15.0",
-    "catalog-converter": "^0.0.2",
+    "catalog-converter": "0.0.2-alpha.2",
```

Comment:
Awaiting catalog-converter PR to increment version

Reply:
catalog-converter PR has been merged, but still has this `0.0.2-alpha.2` version

Reply:
Update it to fit, I haven't been able to test new version